pkg/api/v1/types.go

```diff
@@ -1241,6 +1243,10 @@ type PodSpec struct {
 	ImagePullSecrets []LocalObjectReference `json:"imagePullSecrets,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
 }
 
+// PodSecurityContext holds pod-level security attributes and common container settings.
+type PodSecurityContext struct {
```

Comment:
I'm still only half following - is this PR ready to commit?  If so, nuke this empty struct and bring it back when you need it.  If not, why?